recipes/multivisor/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "multivisor" %}
+{% set version = "6.0.1" %}
+
+package:
+  name: multivisor-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/multivisor-{{ version }}.tar.gz
+  sha256: f834f7b6968d478e1e2e24638d4fc29a9a57947fe5642cb266f109745112835b
+
+build:
+  number: 0
+
+outputs:
+  - name: multivisor
+    script: build_core.sh
+    build:
+      noarch: python
+      entry_points:
+        - multivisor=multivisor.server.web:main
+        - multivisor-rpc=multivisor.server.rpc:main
+        - multivisor-cli=multivisor.client.cli:main
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - gevent >=1.3
+        - python >=3.6
+        - six
+    test:
+      imports:
+        - multivisor
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: multivisor-web
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage('multivisor', exact=True) }}
```

Comment:
I think there were some issues with `exact=True`, multi output recipes and noarch, which might require you to use `max_pin='x.x.x.x'`, instead, but I can't recall what was the problem :/

Reply:
I think I had to put the same host requirement everywhere to get the same build string.
I don't see any issue using max_pin. I changed to that.